charts/console/values.yaml

```diff
@@ -234,6 +234,13 @@ secret:
       # tlsCert:
       # tlsKey:
 
+# -- Settings for license key, as an alternative to secret.enterprise when
+# a license secret is available
```

Comment:
Maybe I don't understand previous intention, but for me secret.enterprise.license is a way to define plain text Redpanda license in values.yaml that is then propagated to secret configuration file 

https://github.com/redpanda-data/helm-charts/blob/eae186334a49c651fbc731dbc0baaa08ecc6d400/charts/console/templates/secret.yaml#L51

Because of that I would suggest to change the comment/documentation to reflect that `secret.enterprise.license` is a plain text representation and will be stored in helm release values. That precise note could be applied in both new `enterprise.licenseSecretRef` value and old `secret.enterprise.license` value.

Reply:
Sure, we can start to make those kind of doc changes that are definitely needed.

Reply:
So it will be addressed in the next PR or this conversation will not have follow up?

Reply:
We can have a followup, this is just the first PR.